web/src/engine/websites/JManga.ts

```diff
@@ -0,0 +1,22 @@
+import { Tags } from '../Tags';
+import icon from './JManga.webp';
+import { DecoratableMangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import * as Liliana from './decorators/Liliana';
+import { pageScript } from './RawOtaku';
+
+@Common.MangaCSS(/^{origin}\/read\/[^/]+-raw\/$/, Liliana.queryMangaTitleFromURI)
+@Common.MangasMultiPageCSS('/raw-manga/?p={page}', Liliana.queryMangas, 1, 1, 0, Common.AnchorInfoExtractor(true))
+@Common.ChaptersSinglePageCSS('div.chapters-list-ul ul li.item a.item-link', Common.AnchorInfoExtractor(true) )
+@Common.PagesSinglePageJS(pageScript, 500)
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
```

Comment:
Is this really a Liliana based website? It looks different and does not import Liliana resources like the other websites.

Reply:
As weird as it is, they share some CSS selectors that are unique like `div.anisc-detail` and others.

Anyway, like you said, liliana is not a decorator just a set of CSS and a script, importing values for it is not a big deal.